blog/views/admin_post_views.py

```
@@ -12,24 +13,46 @@ def admin_post_list_view(request):
     return HttpResponse("관리자 게시물 목록 페이지")
 
 
+# 관리자 게시물 생성
+@staff_member_required
+def admin_post_list_view(request):
```

Ruff (0.15.5)

[warning] Unused function argument: `request`

(ARG001)